xendit-java-library-example/src/main/java/ExampleWithClient/ExampleGetPHPDisbursement.java

```diff
@@ -0,0 +1,37 @@
+package ExampleWithClient;
+
+import com.xendit.exception.XenditException;
+
+import java.util.Arrays;
+
+import com.xendit.XenditClient;
+import com.xendit.model.DisbursementPHP;
+
+public class ExampleGetPHPDisbursement {
+    public static void main(String[] args) {
+        // create xendit client which holds value of apikey
+        XenditClient xenditClient = new XenditClient.Builder()
+                .setApikey("xnd_development_nsn3PhwpWyMZeCZ9YZNlzh5TVmGTqYxVBoXpdbz2glHi1Nk1dBZOibZdUmqw7")
```

Comment:
@hglohika please put a dummy for this one instead.

Reply:
72b94b28cc3026095e2f0bdc0755e4fc1a00aa2f